cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -0,0 +1,408 @@
+import unittest
+import pandas as pd
+from cellxgene_schema import ontology
+from cellxgene_schema import validate
+import numpy
+import fixtures.examples_validate as examples
+from scipy import sparse
+
+
+class TestFieldValidation(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.schema_def = validate._get_schema_definition(examples.SCHEMA_VERSION)
+        cls.OntologyChecker = ontology.OntologyChecker()
+
+    def setUp(self):
+        self.validator = validate.Validator()
+        self.validator.adata = examples.adata_empty
+        self.column_name = "cell_type_ontology_term_id"
+        self.column_schema = self.schema_def["components"]["obs"]["columns"][
+            self.column_name
+        ]
+        self.curie_constraints = self.schema_def["components"]["obs"]["columns"][
+            self.column_name
+        ]["curie_constraints"]
+
+    def test_schema_defintion(self):
+        """
+        Tests that the definition of schema is correct
```

Comment:
```suggestion
        Tests that the definition of schema is well-formed
```

I'm worried that "correct" might imply that it meets the written 2.0.0 spec, while, I believe that this test is just checking the structure of the schema file can be parsed. Is that correct?